src/main/java/com/clouway/chita/HttpClient.java

```diff
@@ -94,7 +94,11 @@
 
         }
 
-        inputStream = conn.getInputStream();
+        try {
```

Comment:
Yeah, error stream is available only when response code is not 200. 

If you check error stream for null you will get the correct result I think. 

200 -> conn.getInputStream(); 
Any other code -> conn.getErrorStream();